docs/source/user_guide/introduction.rst

```diff
@@ -41,8 +41,8 @@ can simply write:
     forecaster = ReducedRegressionForecaster(regressor, window_length=12)
     forecaster.fit(y_train)
     y_pred = forecaster.predict(fh)
-    smape_loss(y_test, y_pred)
-    >>> 0.12726230426056875
+    SymmetricMeanAbsolutePercentageError(y_test, y_pred)
+    >>> 12.574429324402637
```

Comment:
Could we scale them so that they are between 0 and 1? I think this is how MAPE is defined in scikit-learn too: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_percentage_error.html

Reply:
MAPE in general won't be between 0 and 1 (it can be greater than 1), but what you mean is that 1 should correspond to 100% percentage error?

Reply:
I saw that in the scikit docs. I'm actually not sure why they mention the 0 to 100 range in their documentation. Bad predictions can definitely have arbitrarily large MAPE values (e.g. prediction is always 4 and true value is 1 would yield a MAPE of 300 or 3, depending on how you scale). Right now I'm showing 100% error as 100 (Hyndman paper referenced in docstring multiplies by 100), but we could just as easily not scale by 100.

Reply:
Yes 1 meaning 100% is what I meant

Reply:
Let's follow scikit-learn rather than the Hyndman paper, I remember there were some inconsistencies in the M4 study too which always required to go back and forth between dividing/multiplying by 100

Reply:
Switched this to make 100 percent error = 1.